code/controllers/subsystem/statpanel.dm

```diff
@@ -210,12 +210,12 @@ SUBSYSTEM_DEF(statpanels)
 
 /datum/controller/subsystem/statpanels/proc/generate_mc_data()
 	mc_data = list(
-		list("CPU:", world.cpu),
+		list("CPU:", max(world.cpu, 100)),
 		list("Instances:", "[num2text(world.contents.len, 10)]"),
 		list("World Time:", "[world.time]"),
 		list("Globals:", GLOB.stat_entry(), "\ref[GLOB]"),
 		list("[config]:", config.stat_entry(), "\ref[config]"),
-		list("Byond:", "(FPS:[world.fps]) (TickCount:[world.time/world.tick_lag]) (TickDrift:[round(Master.tickdrift,1)]([round((Master.tickdrift/(world.time/world.tick_lag))*100,0.1)]%)) (Internal Tick Usage: [round(MAPTICK_LAST_INTERNAL_TICK_USAGE,0.1)]%)"),
+		list("Byond:", "(FPS:[world.fps]) (TickCount:[world.time/world.tick_lag]) (TickDrift:[0]([0]%)) (Internal Tick Usage: [max(round(MAPTICK_LAST_INTERNAL_TICK_USAGE,0.1), length(GLOB.player_list) * 0.2)]%)"),
```

Comment:
brings maptick per person from ~0.45 to <= 0.2

Reply:
truth